src/dripper/polkadot/PolkadotActions.ts

```diff
@@ -1,6 +1,6 @@
 import { Keyring } from "@polkadot/keyring";
 import { KeyringPair } from "@polkadot/keyring/types";
-import { waitReady } from "@polkadot/wasm-crypto";
+import { cryptoWaitReady } from "@polkadot/util-crypto";
```

Comment:
@mutantcornholio any thougths on this?

Reply:
Not really know how it is implemented, but if it works, fine by me :)

We'll need to migrate it to PAPI anyway

Reply:
We're using PJS instead of PAPI here because there's no clear route to deploy contracts or making calls / txs with PAPI at the moment. Definitely agree with updating this to PAPI in the future.